packages/ui/src/pages/Home/MultisigView.tsx

```diff
@@ -4,52 +4,55 @@ import MultisigActionMenu from './MultisigActionMenu'
 import { styled } from '@mui/material/styles'
 import { Box, Chip } from '@mui/material'
 import { useMultiProxy } from '../../contexts/MultiProxyContext'
-import Balance from '../../components/library/Balance'
-import { renderMultisigHeading } from '../multisigHelpers'
 import MultisigAccordion from './MultisigAccordion'
+import { Balance } from '../../components/library'
 
 const MultisigView = () => {
-  const { selectedMultiProxy, selectedHasProxy, selectedIsWatched } = useMultiProxy()
+  const { selectedMultiProxy, selectedHasProxy } = useMultiProxy()
 
   return (
     <MultisigViewWrapperStyled>
       <HeaderStyled>
-        {selectedMultiProxy && (
-          <h3>
-            {selectedHasProxy
-              ? 'Controlled by'
-              : renderMultisigHeading(selectedMultiProxy.multisigs.length > 1)}
-          </h3>
-        )}
-        {!selectedHasProxy && (
+        {selectedMultiProxy && <h3>{selectedHasProxy ? 'Controlled by' : 'Account Details'}</h3>}
+        {selectedHasProxy && (
           // only show the buttons if we have a solo multisig
           // and only show the send button if we are part of the multisig
           <BoxStyled>
-            <MultisigActionMenu withSendButton={!selectedIsWatched} />
+            <MultisigActionMenu
+              menuButtonBorder={'none'}
+              withNewTransactionButton={false}
+            />
```

Comment:
Unless I'm mistaking, we should actually never have this action buttons any more. At least not now. Maybe we should comment it out.